warmer.rb

```diff
@@ -34,8 +37,8 @@ def authorizer
     )
   end
 
-  def config
-    @config ||= YAML.load(ENV['CONFIG'])
+  def pools
+    @pools ||= redis.smembers('warmerpools').map { |pool| JSON.parse(pool) }
```

Comment:
One thing worth considering here is that since it is cached in memory, updates will only propagate after process restart. Maybe it's not a huge problem, but I could certainly see that being confusing to an operator.

We could maybe have a timer that resets every minute.